Makefile

```diff
@@ -465,16 +465,15 @@ endif
 ifeq ($(OS), Darwin)
 	-cat $(JULIAHOME)/contrib/mac/startup.jl >> $(DESTDIR)$(prefix)/etc/julia/startup.jl
 endif
-
 ifeq ($(OS), WINNT)
 	cd $(BUILDROOT)/julia-$(JULIA_COMMIT)/bin && rm -f llvm* llc.exe lli.exe opt.exe LTO.dll bugpoint.exe macho-dump.exe
+endif
+	cd $(BUILDROOT) && $(TAR) zcvf $(JULIA_BINARYDIST_FILENAME).tar.gz julia-$(JULIA_COMMIT)
 
+exe:
```

Comment:
Looks like I forgot `exe: binary-dist`

Reply:
Do we really want to slow the build process down by adding this? FWIW copying all the files back on my computer takes 2-5 minutes

Reply:
Yes, we do.  It's better to haev a slow build to create something we know is correct, then for there to be a possibility to have old files shipped.